frontend/src/metabase/writeback/containers/WritebackForm.tsx

```diff
@@ -13,7 +13,7 @@ import CategoryFieldPicker from "./CategoryFieldPicker";
 export interface WritebackFormProps {
   table: Table;
   row?: unknown[];
-  onSubmit?: () => void;
+  onSubmit?: (values: { [key: string]: number | string }) => void;
```

Comment:
I think it'd make sense to use `unknown` or [`Value` type](https://github.com/metabase/metabase/blob/22923384ccc54c5e0fb879f2fcf0dbdd0883efc5/frontend/src/metabase-types/types/Dataset.ts#L31) instead of `number | string` as we're going to have other types too